app/frontend/src/pages/layout/Layout.module.css

```diff
@@ -82,11 +82,6 @@
     display: none;
 }
 
-.microsoftLogo,
```

Comment:
I removed the files and CSS rules because they're no longer used, these were removed in @zedhaque's PR a while ago that improved responsive design.